NEWS.rst

```diff
@@ -8,6 +8,131 @@ https://twisted.org/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.8.0.rc1 (2022-08-28)
+===============================
+
+Features
+--------
+
+- twisted.internet.defer.maybeDeferred will now schedule a coroutine result as asynchronous operation and return a Deferred that fires with the result of the coroutine. (#10327)
+- Twisted now works with Cryptography versions 37 and above, and as a result its minimum TLS protocol version has been upgraded to TLSv1.2. (#10377)
+
+
+Bugfixes
+--------
+
+- ``twisted.internet.base.DelayedCall.__repr__`` will no longer raise ``AttributeError`` if the ``DelayedCall`` was created before debug mode was enabled.  As a side-effect, ``twisted.internet.base.DelayedCall.creator`` is now defined as ``None`` in cases where previously it was undefined. (#8306)
+- twisted.internet.iocpreactor.udp now properly re-queues its listener when there is a failure condition on the read from the socket. (#10052)
+- twisted.internet.defer.inlineCallbacks no longer causes confusing StopIteration tracebacks to be added to the top of tracebacks originating in triggered callbacks (#10260)
+- The typing of twisted.internet.task.react no longer constrains the type of argv. (#10289)
+- `ContextVar.reset()` now works correctly inside `inlineCallbacks` functions and coroutines. (#10301)
+- Implement twisted.python.failure._Code.co_positions for compatibility with Python 3.11. (#10336)
+- twisted.pair.tuntap._TUNSETIFF and ._TUNGETIFF values are now correct parisc, powerpc and sparc architectures. (#10339)
+
+
+Improved Documentation
+----------------------
+
+- The release process documentation was updated to include information about
+  doing a security release. (#10324)
+- The development and policy documentation pages were moved into the same
+  directory that is now placed inside the documentation root directory. (#11575)
+
+
+Deprecations and Removals
+-------------------------
+
+- Python 3.6 is no longer supported.
+  Twisted 22.4.0 was the last version with support for Python 3.6. (#10304)
+
+
+Misc
+----
+
+- #9437, #9495, #10066, #10275, #10318, #10325, #10328, #10329, #10331, #10349, #10350, #10352, #10353, #11561, #11564, #11567, #11569, #11585, #11592, #11600, #11606, #11610, #11612, #11614
+
+
+Conch
+-----
+
+Bugfixes
+~~~~~~~~
+
+- twisted.conch.checkers.UNIXAuthorizedKeysFiles now uses the filesystem encoding to decode usernames before looking them up in the password database, so it works on Python 3. (#10286)
```

Comment:
This should be reverted and removed from the release

see https://github.com/twisted/twisted/issues/10286